src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,46 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+### Deprecation Notice
+- Support for Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) has been deprecated and will be removed next release.
+
+### Features
+
+- Update okteto cli version to [2.13.0](https://github.com/okteto/okteto/releases/tag/2.13.0)
+- Default ingress class for ingresses managed by okteto is `okteto-nginx`. All ingresses managed by okteto are migrated automatically during the upgrade
```

Comment:
I don't think we need more docs for this.
@jpf-okteto maybe here, instead of the sameple we have, we can say what's our default nginx configuration (the whole yaml):
https://www.okteto.com/docs/self-hosted/administration/configuration/#nginx-ingress

Reply:
Done in this PR: https://github.com/okteto/docs/pull/343